### ISSUE TITLE
Provide a means (tag) to fetch http query parameters

### ISSUE THREAD

**Issue by andrazk** [modxbot/migrate#6653]
andrazk created Redmine issue ID 6653

There are many cases, where you send some data via query parameters to the landing page. There's no means to obtain individual parameter values, unless you write a snippet to get the parameters from $_GET['param'].

http://domain.tld/page/request.html?foo=bar

The snippet I use "getParam" looks like this:

return (isset($_GET[$param])) ? $_GET[$param] : '';

And the call [[!getParam? &param=`foo`]] would return "bar"

Basically, while parsing the contents of the document shown, there could be placeholders or a buil-in modifier to provide access to parameter values.

[[?foo]] could return the value "bar". 
Other option could be a built-in modifier, output filter, [[+modx:queryParam=`foo`]]

**Comment by andrazk** [comment]
andrazk submitted:

There are many cases, where you send some data via query parameters to the landing page. There's no means to obtain individual parameter values, unless you write a snippet to get the parameters from @$_GET['foo']@.

Example:

<pre>
http://domain.tld/page/request.html?foo=bar
</pre>

The snippet I use "getParam" looks like this:

<pre>
return (isset($_GET[$param])) ? $_GET[$param] : '';
</pre>

And the call @[[!getParam? &param=`foo` ]]@ would return "bar"

Basically, while parsing the contents of the document shown, there could be placeholders or a buil-in modifier to provide access to parameter values.

@[[?foo]]@ could return the value "bar". 
Other option could be a built-in modifier, output filter, @[[+modx:queryParam=`foo`]]@
